app/controllers/cart_controller.rb

```diff
@@ -0,0 +1,5 @@
+class CartController < ApplicationController
+  def show
+    @product = Product.all
```

Comment:
chỗ này em lại lấy toàn bộ các cột dữ liệu rồi, như thế ko ổn về hiệu năng
Anh có nhắc nhiều rồi, em sửa lại nhé, nếu cần thì thêm cả phần sắp xếp thứ tự vào nha